app/uk/gov/hmrc/pushpullnotificationsgateway/models/OutboundNotification.scala

```diff
@@ -16,5 +16,4 @@
 
 package uk.gov.hmrc.pushpullnotificationsgateway.models
 
-case class ForwardedHeader(key: String, value: String)
-case class OutboundNotification(destinationUrl: String, forwardedHeaders: List[ForwardedHeader], payload: String)
+case class OutboundNotification(destinationUrl: String, payload: String)
```

Comment:
shouldn't this be "case class OutboundNotification(destinationUrl: String, payload: NotificationResponse)"? Then the connector would automatically add the json content type

Reply:
Heh, there wasn't a `NotificationResponse` type in Gateway...